README.md

```diff
@@ -59,6 +59,9 @@ In particular, if the closure is `|_| false` or `T` is uninhabited, then this co
 Similarly, the function `predict<T>(fn(T) -> bool) -> Nondet<T>` also returns a `T` satisfying the closure, but this non-determinism is interpreted *angelically*, which means there has to *exist* a possible choice that makes the program behave as intended.
 In particular, if the closure is `|_| false` or `T` is uninhabited, then this operation is exactly the same as `hint::unreachable_unchecked()`: no possible choice exists, and hence ever reaching this operation is Undefined Behavior.
 
+For the purpose of making the spec executable, `pick` also receives a `Distribution` argument.
+This argument does not affect the set of possible program behaviors, it is purely a hint for the interpreter.
```

Comment:
This should be with the other `pick` discussion above. Also the signature given in the text is still wrong.